vars/fhBuildNode.groovy

```diff
@@ -2,9 +2,11 @@
 
 def call(Map parameters = [:], body) {
 
-    def label = parameters.get('label', 'nodejs-ubuntu')
+    def labels = ['nodejs', 'ubuntu']
+    labels += parameters.get('labels',[])
+    labels += [parameters.get('label')]
```

Comment:
Looking at this again, I think the logic is wrong. This will always add the "nodejs" and "ubuntu" labels, where previously we were overriding the label if one was passed in. This will break a few of our Jenkinsfiles where we explicitly set a node label.